### PR TITLE
Added pre-commit

### PR DESCRIPTION
**Pull request by andrii-ivaniuk**
Added formatter and pre-commit

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -57,7 +61,6 @@ def read_version():
         'Issue Tracker': 'https://github.com/MasterSergius/acachecontrol/issues',
     },
     python_requires='>=3.6',
-    install_requires=[
-        'aiohttp'
-    ]
+    tests_require=test_requires,
+    install_requires=requires,
```
Comment (MasterSergius) [inline setup.py:59]: IMHO, this should follow this strategy https://packaging.python.org/discussions/install-requires-vs-requirements/

File: setup.py [hunk 2]
```diff
@@ -23,6 +18,15 @@ def read_version():
                 return line.split('=')[1].strip().strip("'")
 
 
+# get the dependencies and installs
+with open("requirements.txt", "r", encoding="utf-8") as f:
+    requires = [x.strip() for x in f if x.strip()]
+
+
+# get test dependencies and installs
+with open("test_requirements.txt", "r", encoding="utf-8") as f:
```
Comment (MasterSergius) [inline setup.py]: How does it affect the package? Package should contain minimal dependencies and should not include tests

Comment (andrii-ivaniuk) [reply setup.py]: https://godatadriven.com/blog/a-practical-guide-to-using-setup-py/

File: setup.py (hunk 1 above)
Comment (andrii-ivaniuk) [reply setup.py:59]: Do we expect to have different dependencies in `setup.py` and `requirements.txt`?

Comment (MasterSergius) [reply setup.py:59]: Whereas install_requires metadata is automatically analyzed by pip during an install, requirements files are not, and only are used when a user specifically installs them using pip install -r.

**Review by MasterSergius — APPROVED** [review]
LGTM

**Pull request merged**